tools/project_manager/page_detail.php

```diff
@@ -80,14 +61,14 @@
     {
         if (is_null($round_for_page_selection) )
         {
-            echo sprintf( _("Showing only the pages of user '%s'."), 
-                          $username_for_page_selection );
+            echo sprintf( _("Showing only the pages of user '%s'."),
+                          html_safe($username_for_page_selection) );
```

Comment:
Are these actually necessary? Given we've validated `$username_for_page_selection` above (forcing it to be NULL or a valid user). `check_username` on registration only allows `/[^-a-zA-Z0-9@. ]/`, none of which are HTML escapes.

Reply:
They aren't now that we're validating the username and round (they were before we did it). I can remove them to simplify the diff if you'd like.

Reply:
Either works for me. I suppose escaping is a belt-and-braces thing. If the username/rounds are ever allowed to include escapes, then we'd be in trouble.